eng/pipelines/coreclr/jitstress2-jitstressregs-armarch.yml

```diff
@@ -0,0 +1,59 @@
+trigger: none
+
+pr: none
+
+schedules:
+- cron: "0 17 * * 6,0"
```

Comment:
I don't want this to be a scheduled job but just a "on-demand". What changes do I need to do to achieve that?

Reply:
Just remove the "schedules" section